### PR TITLE
Refs #26803 - pin patternfly-react to 2.31.x

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
RPM PR: https://github.com/theforeman/foreman-packaging/pull/3797

**Comment by mmoll** [comment]
[test foreman]
[test katello]

**Comment by mmoll** [comment]
I propose this for 1.22 to unbreak Travis tests and potentially ease cherry-picks later.

File: webpack/assets/javascripts/react_app/components/Pagination/__snapshots__/Pagination.test.js.snap [hunk 1]
```diff
@@ -3,6 +3,8 @@
 exports[`Pagination rendering from erb renders layout 1`] = `
 <Paginator
   className="col-md-12"
+  disableNext={false}
```
Comment (Ron-Lavi) [inline webpack/assets/javascripts/react_app/components/Pagination/__snapshots__/Pagination.test.js.snap]: I am not sure, but wasn't this snapshot already changed in one of the last upgrades? :stuck_out_tongue_closed_eyes:

**Review by Ron-Lavi — APPROVED** [review]

File: webpack/assets/javascripts/react_app/components/Pagination/__snapshots__/Pagination.test.js.snap (hunk 1 above)
Comment (mmoll) [reply webpack/assets/javascripts/react_app/components/Pagination/__snapshots__/Pagination.test.js.snap]: also for 1.22?

**Comment by tbrisker** [comment]
I'm a bit reluctant to upgrade this at this stage of the release process, as it would mean we GA with a different version than the RCs are tested with, and that plugins will likely need to be rebuilt as well. Instead, I'd prefer pinning the package.json in the stable branch to the specific version we have packaged for that branch (which should hopefully also fix the travis failure).
And hopefully by 1.23 we'll have a better way of handling these...

**Comment by mmoll** [comment]
@tbrisker Aw, I forgot about the RPM webpack dance... will replace this with a pinning.

**Comment by mmoll** [comment]
there we go... :gift_heart:

**Review by tbrisker — APPROVED** [review]
Thanks @mmoll !

**Pull request merged**